FakeItEasy.sln

```diff
@@ -1,26 +1,32 @@
-﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 2013
-VisualStudioVersion = 12.0.31101.0
+# Visual Studio 14
+VisualStudioVersion = 14.0.24720.0
 MinimumVisualStudioVersion = 10.0.40219.1
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution Items", "{A6A7B553-5359-4FB9-B19C-8A23004F49DB}"
 	ProjectSection(SolutionItems) = preProject
 		.editorconfig = .editorconfig
-		CommonAssemblyInfo.cs = CommonAssemblyInfo.cs
-		FakeItEasy.Dictionary.Tests.xml = FakeItEasy.Dictionary.Tests.xml
+		.gitattributes = .gitattributes
+		.gitignore = .gitignore
+		.mailmap = .mailmap
+		CONTRIBUTING.md = CONTRIBUTING.md

```

Comment:
In my understanding, the idea is that the solution is effectively the "root" of the entire thing, so if you open the solution, you can effectively edit anything and everything from inside VS.